library/timing.c

```diff
@@ -263,6 +263,27 @@ unsigned long mbedtls_timing_get_timer( struct mbedtls_timing_hr_time *val, int
     }
 }
 
+unsigned long long mbedtls_timing_get_timer_us( struct mbedtls_timing_hr_time *val, int reset )
```

Comment:
This duplicates the code of the existing function `mbedtls_timing_get_timer`. `mbedtls_timing_get_timer` should be rewritten as a wrapper around `mbedtls_timing_get_timer_us`, both for the Windows and the Unix implementation.